README.md

```diff
@@ -32,7 +32,7 @@ Install the package:
 $ yarn add react-native-config
 ```
 
-Link the library:
+Link the library (not supported on Windows, use manual linking):
```

Comment:
we added autolinking to rnw on 63 (?) @jonthysell to confirm

Reply:
It did not work for me. I've opened an issue for that: https://github.com/microsoft/react-native-windows/issues/6024